### PR TITLE
dev into main

### PR DESCRIPTION
**Pull request by mlAgostini**
dev into main

### CODE REVIEW

File: frontend/src/App.jsx [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
-
-import "./App.css";
+import Filter from "./components/Filter/Filter";
+import "./styles/App.css";
 
 function App() {
   return (
-    <div className="App">
+    <>
+      {/* Bouton du panier ICIIIIIIII */}
```
Comment (fantinerudent) [inline frontend/src/App.jsx]: no comment plz

File: frontend/src/components/CardHouseware/CardHouseware.jsx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+import React from "react";
+import Stars from "../Stars/Stars";
+
+/* eslint-disable no-param-reassign */
+function CardHouseware({ note, houseware, handleAddArticle }) {
+  return (
+    <div className="carte">
+      {/* <h5>name</h5>
```
Comment (fantinerudent) [inline frontend/src/components/CardHouseware/CardHouseware.jsx]: no comment lines

File: frontend/src/components/z-old/Card/Card.jsx [hunk 3]
```diff
@@ -0,0 +1,15 @@
+/* import SCard from "./CardStyle";
```
Comment (fantinerudent) [inline frontend/src/components/z-old/Card/Card.jsx]: delete files

File: frontend/src/components/z-old/Card/CardStyle.jsx
Comment (fantinerudent) [inline frontend/src/components/z-old/Card/CardStyle.jsx:1]: delete file

File: frontend/src/main.jsx [hunk 4]
```diff
@@ -1,12 +1,16 @@
 import React from "react";
 import ReactDOM from "react-dom/client";
-
+import "./styles/App.css";
+import { BrowserRouter } from "react-router-dom";
+// import Home from "./pages/Home";
```
Comment (fantinerudent) [inline frontend/src/main.jsx]: no cmt plz

File: frontend/src/pages/Home.jsx [hunk 5]
```diff
@@ -1,36 +1,38 @@
-import Counter from "../components/Counter";
-import logo from "../assets/logo.svg";
+import { useState } from "react";
+import Cart from "@components/SideBar/Cart";
+import { Route, Routes } from "react-router-dom";
+import Navbar from "../components/Navbar/Navbar";
+// import CardHouseware from "../components/CardHouseware/CardHouseware";
```
Comment (fantinerudent) [inline frontend/src/pages/Home.jsx]: no cmt plz

File: frontend/src/services/api-calls.js [hunk 6]
```diff
@@ -0,0 +1,21 @@
+const baseUrl = "https://acnhapi.com/v1a";
+
+/* function getFishs() {
```
Comment (fantinerudent) [inline frontend/src/services/api-calls.js]: delete cmt

File: frontend/src/styles/Cart.css [hunk 7]
```diff
@@ -0,0 +1,95 @@
...
+  display: flex;
+  justify-content: right;
+  align-items: center;
+  z-index: 2;
+}
+
+.modal-content {
+  display: flex;
+  flex-direction: column;
+  /* justify-content: center; */
```
Comment (fantinerudent) [inline frontend/src/styles/Cart.css]: cmt

**Review by fantinerudent — CHANGES_REQUESTED** [review]
Delete old files & comments plz

File: frontend/src/App.jsx (hunk 1 above)
Comment (fmarincode) [reply frontend/src/App.jsx]: done

File: frontend/src/components/CardHouseware/CardHouseware.jsx (hunk 2 above)
Comment (fmarincode) [reply frontend/src/components/CardHouseware/CardHouseware.jsx]: done

File: frontend/src/components/z-old/Card/Card.jsx (hunk 3 above)
Comment (fmarincode) [reply frontend/src/components/z-old/Card/Card.jsx]: deleted

File: frontend/src/components/z-old/Card/CardStyle.jsx
Comment (fmarincode) [reply frontend/src/components/z-old/Card/CardStyle.jsx:1]: deleted

File: frontend/src/main.jsx (hunk 4 above)
Comment (fmarincode) [reply frontend/src/main.jsx]: done

File: frontend/src/pages/Home.jsx (hunk 5 above)
Comment (fmarincode) [reply frontend/src/pages/Home.jsx]: done

File: frontend/src/services/api-calls.js (hunk 6 above)
Comment (fmarincode) [reply frontend/src/services/api-calls.js]: deleted + old url acnhapi too

File: frontend/src/styles/Cart.css (hunk 7 above)
Comment (fmarincode) [reply frontend/src/styles/Cart.css]: done

**Review by fantinerudent — APPROVED** [review]

**Review by fmarincode — APPROVED** [review]

**Pull request merged**